apps/server/src/infra/sync/strategy/tsp/tsp-legacy-migration.service.integration.spec.ts

```diff
@@ -1,13 +1,13 @@
 import { createMock, DeepMocked } from '@golevelup/ts-jest';
 import { EntityManager } from '@mikro-orm/mongodb';
+import { SystemType } from '@modules/system';
 import { Test, TestingModule } from '@nestjs/testing';
 import { SchoolEntity } from '@shared/domain/entity';
 import { SystemProvisioningStrategy } from '@shared/domain/interface/system-provisioning.strategy';
 import { SchoolFeature } from '@shared/domain/types';
 import { cleanupCollections, schoolEntityFactory, systemEntityFactory } from '@shared/testing';
 import { Logger } from '@src/core/logger';
-import { MongoMemoryDatabaseModule } from '@src/infra/database';
-import { SystemType } from '@src/modules/system';
+import { MongoMemoryDatabaseModule } from '../../../database';
```

Comment:
Not the same module.
```suggestion
import { MongoMemoryDatabaseModule } from '@src/infra/database';
```